models/chainlink/optimism/chainlink_optimism_price_feeds.sql

```diff
@@ -4,11 +4,11 @@
     materialized = 'incremental',
     file_format = 'delta',
     incremental_strategy = 'merge',
-    unique_key = ['blockchain', 'block_time', 'proxy_address', 'oracle_price', 'aggregator_address', 'underlying_token_address'],
+    unique_key = ['blockchain', 'block_number', 'proxy_address'],
```

Comment:
I wonder if we'd need 'aggregator_address' here? Not sure if there's ever a case where a proxy is used multiple times.

Reply:
hey @MSilb7 ! yes we can remove the aggregator_address for the unique_key, but will have to add in the underlying_token_address as there can be >1 token using the same feed